full_snapshot.proto

```diff
@@ -235,6 +236,61 @@ message TablespaceInformation {
   repeated string config = 4;   // Tablespace-level options, as "keyword=value" strings
 }
 
+message CollectorConfig {
+  string section_name = 1;
+
+  bool disable_logs = 2;
+  bool disable_activity = 3;
+  bool enable_log_explain = 4;
+
+  string db_name = 14;
+  string db_username = 15;
+  string db_host = 16;
+  int32 db_port = 17;
+  string db_sslmode = 18;
+  string db_sslrootcert_path = 19;
+  string db_sslcert_path = 20;
+  string db_sslkey_path = 21;
```

Comment:
As mentioned (see my other comment before this change), I think we can actually do `has_` here as well:

```
  bool has_db_sslrootcert_path = 19;
  bool has_db_sslcert_path = 20;
  bool has_db_sslkey_path = 21;

Reply:
And same for `azure_ad_certificate_path` below, right?